### DISCUSSION TITLE
picodet_lcnet_x1_0_layout.yml更换模型#paddleocr更换模型

### DISCUSSION THREAD

**Discussion by Andy197527** [PaddlePaddle/PaddleOCR#13223] [Q&A] [answered]
#### 问题描述 / Problem Description
一个paddleocr的版面分析模型，就是如何把picodet_lcnet_x1_0_layout.yml这个训练好的模型给换到paddleocr里边去？具体要怎么操作呢？

#### 运行环境 / Runtime Environment
- OS:
- Paddle:2.6
- PaddleOCR:

**Comment by GreatV** [comment]
请参考 https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppstructure/docs/inference.md#2-%E5%85%B3%E9%94%AE%E4%BF%A1%E6%81%AF%E6%8A%BD%E5%8F%96

**Reply by Andy197527** [reply]
> 请参考 https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppstructure/docs/inference.md#2-%E5%85%B3%E9%94%AE%E4%BF%A1%E6%81%AF%E6%8A%BD%E5%8F%96

但是我想在下载好的Paddleocr里边更换，而不是重新再下载了，有没有别的替换方式，就是再python环境下边替换

**Comment by 77bw** [comment, marked as answer]
请问有替换模型的方式了吗

**Reply by SWHL** [reply]
如果只想推理的话，可以用我整理的[RapidLayout](https://github.com/RapidAI/RapidLayout)

**Reply by Andy197527** [reply]
> 请问有替换模型的方式了吗

你找到了吗？我还是没找到

**Reply by Andy197527** [reply]
> 如果只想推理的话，可以用我整理的[RapidLayout](https://github.com/RapidAI/RapidLayout)

我想替换掉原先的模型，使用我自己训练好的模型
